test/functionalTest/cases/0953_list_all_ents_id/GET_v2_entities_type.test

```diff
@@ -0,0 +1,216 @@
+      "type": "T3"
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --json
+echo
+echo
+
+
+echo "02. GET /v2/entities?type=T1"
+echo "============================"
+orionCurl --url '/v2/entities?type=T1' --json
+echo
+echo
+
+
+echo "03. GET /v2/entities?type=T3"
+echo "============================"
+orionCurl --url '/v2/entities?type=T3' --json
+echo
+echo
+
+
+echo "04. GET /v2/entities?type=NONE"
+echo "=============================="
+orionCurl --url '/v2/entities?type=NONE' --json
+echo
+echo
+
+
+--REGEXPECT--
+01. POST /v1/updateContext
+==========================
+HTTP/1.1 200 OK
+Content-Length: 1072
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T1"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E2",
+                "isPattern": "false",
+                "type": "T1"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E3",
+                "isPattern": "false",
+                "type": "T3"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T3"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E2",
+                "isPattern": "false",
+                "type": "T3"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+
+
+02. GET /v2/entities?type=T1
+============================
+HTTP/1.1 200 OK
+Content-Length: 49
+Content-Type: application/json
+Date: REGEX(.*)
+
+[
+    {
+        "id": "E1",
+        "type": "T1"
+    },
+    {
+        "id": "E2",
+        "type": "T1"
+    }
+]
+
+
+03. GET /v2/entities?type=T3
+============================
+HTTP/1.1 200 OK
+Content-Length: 73
+Content-Type: application/json
+Date: REGEX(.*)
+
+[
+    {
+        "id": "E3",
+        "type": "T3"
+    },
+    {
+        "id": "E1",
+        "type": "T3"
+    },
+    {
+        "id": "E2",
+        "type": "T3"
+    }
+]
+
+
+04. GET /v2/entities?type=NONE
+==============================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Date: REGEX(.*)
+
+[]
+
+
+--TEARDOWN--
+brokerStop CB
+dbDrop CB
+
```

Comment:
Yes!
And also, many smaller tests instead of *one big one* ...
Very good indeed!

NTC